Themes/default/scripts/profile.js

```diff
@@ -293,4 +293,32 @@ function changeVariant(iThemeId, el)
 	document.getElementById('theme_thumb_' + iThemeId).src = oThemeVariants[iThemeId][el.value]['thumbnail'];
 	document.getElementById('theme_thumb_preview_' + iThemeId).href = el.form.action + ';theme=' + iThemeId + ';variant=' + el.value;
 	document.getElementById('theme_preview_' + iThemeId).href = el.form.action + ';theme=' + iThemeId + ';variant=' + el.value;
-}
\ No newline at end of file
+}
+
+$(document).on('change', '#export_format_select', function() {
+	var selected_format = $('#export_format_select').val();
+	if (completed_formats.includes(selected_format))
```

Comment:
Array.prototype.includes  is ES6 syntax, we might have problems with old browsers compatibility.

Reply:
Do you happen to have a quick suggestion for a backwards compatible alternative? 

(Don't spend a bunch of time answering this; I can do my own homework if I need to. But if you can toss off an answer in your sleep, I won't mind saving some time.)

Reply:
Theres jquery's $.inArray  or vanilla indexOf

Reply:
Thanks